src/tests/ut/doctor/doctor_test.cpp

```diff
@@ -1,17 +1,105 @@
 #include "doctor_test.hpp"
-
+#include "gender.hpp"
 namespace
 {
 
-TEST_F(DoctorTestFixture, test)
+class DoctorAppendVisitFunction : public DoctorTestFixture
+{
+};
+
+class DoctorRemoveVisitFunction : public DoctorTestFixture
+{
+};
+
+TEST_F(DoctorTestFixture, CreatingDoctorViaCreateShouldAppendDoctorsToClinic)
 {
     Doctor::createDoctor("Jan", "Kowalski", "80062378532", Gender::Male);
     Doctor::createDoctor("Janusz", "Schmit", "80062378532", Gender::Male);
     Doctor::createDoctor("Gawel", "Pawel", "80062378532", Gender::Male);
     Doctor::createDoctor("Jan", "Poniatowski", "80062378532", Gender::Male);
-    auto expected_doctor = Clinic::getDoctors().at(3);
 
-    EXPECT_EQ(Doctor::getDoctor("Jan", "Poniatowski").front(), expected_doctor);
+    auto first_expected_doctor = Clinic::getDoctors().at(0);
+    auto second_expected_doctor = Clinic::getDoctors().at(1);
+    auto third_expected_doctor = Clinic::getDoctors().at(2);
+    auto fourth_expected_doctor = Clinic::getDoctors().at(3);
+
+    EXPECT_EQ(Doctor::getDoctor("Jan", "Kowalski").front(), first_expected_doctor);
+    EXPECT_EQ(Doctor::getDoctor("Janusz", "Schmit").front(), second_expected_doctor);
+    EXPECT_EQ(Doctor::getDoctor("Gawel", "Pawel").front(), third_expected_doctor);
+    EXPECT_EQ(Doctor::getDoctor("Jan", "Poniatowski").front(), fourth_expected_doctor);
+}
```

Comment:
Two doctors would be enough ;)

Reply:
Yeap. But there were four doctors already. I only decided that as if there are 4, then better to check them all.
All this test is strange a bit as it tests Visit and Clinic even more than Doctor class itself.